packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -386,3 +399,63 @@ export async function sideloadAddIn(
     throw err;
   }
 }
+
+function isDotnetInstalled(): boolean {
+  try {
+    // Find the .Net runtimes installed
+    let result = childProcess.execSync("dotnet --list-runtimes");
+    const pattern = /(?<=Microsoft.NETCore.App )[\d.]+/g;
+    const matches = result.toString("utf-8").match(pattern);
+    let foundDotNet = false;
+
+    // Look for version 5 or greater
+    matches?.forEach((match) => {
+      const major: number = parseInt(match.split(".")[0]);
+      foundDotNet = foundDotNet || major >= 5;
+    });
+
+    return foundDotNet;
+  } catch (err) {
+    return false;
+  }
+}
+
+async function convertJsonToXmlManifest(manifestPath: string): Promise<string> {
+  return new Promise((resolve, reject) => {
+    if (manifestPath.endsWith(".json") && fs.existsSync(manifestPath)) {
+      console.log("Converting json to back compat xml");
```

Comment:
Does the client need to be aware we are doing this conversion? I feel we don't need to log that this is happening

Reply:
It's getting logged out in the debug console . . . I'm not sure it will be output when used in the template, but I think it would.

It would be nice to keep this aspect under the covers and hidden . . . however, if there is something wrong with the json that breaks the conversion process that's how well know what to report.  If we completely hid this we'll have to come up with some way of helping developers know to fix their json.